script/build

```diff
@@ -157,5 +158,8 @@ if (!argv.generateApiDocs) {
       } else {
         console.log('Skipping installation. Specify the --install option to install Atom'.gray)
       }
+    }).then(()=>{
+        // record the versions of binaries that were used to create the successful build
+        generatePackageJsonWithTracking()
```

Comment:
If we do this in `script/build`, it will update `package.json` and invalidate all the caches. It's tricky for our CI.

See my comment above, but to reiterate: Can this config/logging go in a file other than `package.json`?